procdiscovery/pkg/inspectors/langdetect.go

```diff
@@ -23,7 +23,8 @@ func (e ErrLanguageDetectionConflict) Error() string {
 }
 
 type inspector interface {
-	Inspect(process *process.Details) (common.ProgramLanguageDetails, bool)
+	Inspect(process *process.Details) (common.ProgrammingLanguage, bool)
+	GetRuntimeVersion(process *process.Details, podIp string) string
```

Comment:
Maybe we should make this a separate interface that inherits this interface? This way we can avoid the empty implementation on all the non supported languages

Reply:
Let's talk on this f2f